File: engine/operations/snapshots.py
"""文件系统快照工具；实现已迁到 system.snapshots。

保留 re-export 一个版本周期，便于按提交回滚。
"""
from ..system.snapshots import (  # noqa: F401
    DEFAULT_BACKUP_DIR,
    backup_dir,
    snapshot_file,
    snapshot_payload,
)

File: crates/ferry-engine/src/operations/snapshots.rs
//! 文件系统快照工具；实现住在 `system::snapshots`。
//!
//! 语义事实源：`engine/operations/snapshots.py`（同样只是 re-export）。

pub use crate::system::snapshots::{
    backup_dir, default_backup_dir, snapshot_file, snapshot_payload,
};
